Resources/config/services.yml

```
@@ -1,6 +1,12 @@
 parameters:
 
 services:
+    Hslavich\OneloginSamlBundle\Controller\SamlController:
+        class: Hslavich\OneloginSamlBundle\Controller\SamlController
+        tags: ['controller.service_arguments']
+        public: true
+        autowire: true
```

```suggestion
        _defaults:
        autowire: true
        autoconfigure: true

    hslavich_onelogin_saml.saml_controller:
        class: Hslavich\OneloginSamlBundle\Controller\SamlController
        tags: ['controller.service_arguments']
        public: true
        autowire: true
```
By doing that, the deprecation warning about autowiring is not triggered